.travis.yml

```diff
@@ -86,15 +95,15 @@ matrix:
             ]
 
 
-        # OSX 10.12 - xcode 8.3 - Python 3.7
+        # OSX 10.12 - xcode 8.3 - Python 3.8
         - os: osx
-          osx_image: xcode10.1
+          osx_image: xcode8.3
           compiler: clang
           env: >
             [
             CXX=clang++ ,
             CC=clang     ,
-            PYTHON_VERSION=3.7   ,
```

Comment:
I would not be ok to remove Python 3.7 from osx

Reply:
Sorry, that was a mistake. Will revert.